aptos-move/aptos-transactional-test-harness/src/aptos_test_harness.rs

```diff
@@ -320,37 +314,50 @@ static PRECOMPILED_APTOS_FRAMEWORK_V2: Lazy<PrecompiledFilesModules> = Lazy::new
         .cloned()
         .collect::<Vec<_>>();
 
-    let mut options = move_compiler_v2::Options {
-        sources: prod_sources,
+    let options = move_compiler_v2::Options {
+        sources: prod_sources.clone(),
         dependencies: vec![],
         named_address_mapping: named_address_mapping_strings.clone(),
         known_attributes: aptos_framework::extended_checks::get_all_attribute_names().clone(),
         language_version: None,
```

Comment:
Why this is `None` here, should it be using `LanguageVersion::default` or will it unwrap on none to default?

Reply:
None is treated as default